root/static/scripts/common/entity.js

```diff
@@ -317,7 +318,7 @@ import formatTrackLength from './utility/formatTrackLength';
         selectionMessage() {
             return ReactDOMServer.renderToStaticMarkup(
                 exp.l('You selected {releasegroup}.', {
-                    releasegroup: this.reactElement({target: '_blank'}),
+                    releasegroup: <DescriptiveLink entity={this} target="_blank" />,
```

Comment:
It doesn't look like `DescriptiveLink` allows passing through `target="_blank"` (and didn't complain since it's not Flow-typed).

Reply:
Hmm. Seems EntityLink defaults to target blank anyway, which is why this works. So removing target.

Reply:
I don't think it does, but we do have some JS that adds it to all links inside a bubble after it opens, so that's probably what's happening:
https://github.com/metabrainz/musicbrainz-server/blob/1039c43d60a99a140cca350c71bb3b22779334cb/root/static/scripts/edit/MB/Control/Bubble.js#L46-L48

Reply:
Would you suggest allowing passing it to DescriptiveLink instead?

Reply:
It might be needed by https://github.com/metabrainz/musicbrainz-server/pull/1128 (even though that's also inside a bubble, it doesn't seem to share the same bubble code linked above). So if you do add it for that, it wouldn't hurt to be explicit here too.